kinova_tracking/pos_kinova_arm_2.py

```diff
@@ -13,19 +13,18 @@
 import random
 
 
-def move_marker_table(labels_markers_list: list[str], c3d_file: c3d, offset: float) -> np.array:
+def move_marker(marker_to_move: str, c3d_file: c3d, offset: np.ndarray, ) -> np.array:
     """
-    This function applies an offet to the marker of the table
-     so that the hinge is displaced at a given distance on the horizontal plane.
+    This function applies an offet to a marker
 
     Parameters
     ----------
-    labels_markers_list : list[str]
-        List of markers labels
+    marker_to_move: int
+        indices of the marker to move
     c3d_file : c3d
         c3d file to move the markers.
-    offset : float
-        Offset to apply to the markers in mm.
+    offset : list[int]
```

Comment:
why not a numpy array ?

Reply:
it is, I forgot to modify the doc string